source/reference/methods.txt

```diff
@@ -1254,6 +1254,13 @@ Sharding Methods
 
        .. versionadded:: 4.4
 
+   * - :upcoming:`sh.disableAutoMerger()
```

Comment:
[question]

What's this directive? I don't believe I've seen this before and don't understand how it differs from `:ref:` in a case like this.

Reply:
Resolved via Slack.